manim/mobject/types/opengl_vectorized_mobject.py

```diff
@@ -614,7 +614,32 @@ def get_curve_functions_with_lengths(
         for n in range(num_curves):
             yield self.get_nth_curve_function_with_length(n)
 
-    def point_from_proportion(self, alpha):
+    def point_from_proportion(self, alpha: float) -> np.ndarray:
+        """Gets the point at a proportion along the mobject.
+
+        Parameters
+        ----------
+        alpha
+            The proportion along the mobject.
+
+        Returns
+        -------
+        np.ndarray
```

Comment:
Is `np.ndarray` necessary here?
I am asking as it is also in the type hints.
But it rendered correctly:
![image](https://user-images.githubusercontent.com/44469195/114261373-7b91b900-99da-11eb-8bfc-58b59d188776.png)

Reply:
Repeating the return type is required